machine-learning/Dockerfile

```diff
@@ -21,8 +21,8 @@ ENV NODE_ENV=production \
   PYTHONDONTWRITEBYTECODE=1 \
   PYTHONUNBUFFERED=1 \
   PATH="/opt/venv/bin:$PATH" \
-  PYTHONPATH=`pwd`
+  PYTHONPATH=/usr/src
 
 COPY --from=builder /opt/venv /opt/venv
 COPY app .
-ENTRYPOINT ["python", "main.py"]
+ENTRYPOINT ["python", "-m", "app.main"]
```

Comment:
What is the benefit of this change?

Reply:
The imports are now modular so the code expects to run as a module. The benefit of modular imports is being able to easily import code from other other models. For instance, if I want to import from an outer `schemas` module, I can do `from ..schemas import ModelType`, where I would otherwise have to do

```
import sys
sys.path.append("..")
from schemas import ModelType
```

And if I have a module name that's the same name as a dependency, there's no ambiguity because it's being imported with `.module_name` instead of `module_name`.